src/localization/en.js

```diff
@@ -117,14 +117,19 @@ export default {
     },
 
     currentBenefits: {
-      hasSection8: {
+      currentBenefits: 'Current Benefits',
+      selectBenefits:  'Select the benefits you currently receive',
+      yes:             'Yes',
+      no:              'No',
```

Comment:
The "yes" and "no" strings are actually provided by `ControlledRadioYesNo` - meaning they're in `formHelpers.js` rather than `CurrentBenefits.js`. So you can remove them from here for now, and we'll make a separate section for formHelpers strings. Otherwise looks good.